Tools/CISupport/ews-app/manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
ews-app doesn't have proper python3 support yet. The service is still running using Python2. It would be moved to Python3 separately. Please exclude this file from this PR.